src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -21,19 +20,18 @@
     private final static Logger.Log LOG = Logger.newLog(ChatGuiFX.class);
 
 
-    private static final double WIDTH = 1000;
-    private static final double HEIGHT = 500;
+    private static final double WINDOW_WIDTH = 1000;
+    private static final double WINDOW_HEIGHT = 500;
 
     private Stage thestage;                         // Holds the scene that user is currently viewing
```

Comment:
nit: some of these comments cause the lines do be a little long (which causes them to wrap and make comparing before and after a little harder). Consider putting them on the lines above the code? Also see my other note on some of them not being necessary.